examples/load-data-via-graph-construction.ipynb

```diff
@@ -235,92 +220,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 32,
-   "metadata": {
-    "id": "zphC5CAjLEQH"
-   },
-   "outputs": [
-    {
-     "data": {
-      "text/html": [
-       "<div>\n",
-       "<style scoped>\n",
-       "    .dataframe tbody tr th:only-of-type {\n",
-       "        vertical-align: middle;\n",
-       "    }\n",
-       "\n",
-       "    .dataframe tbody tr th {\n",
-       "        vertical-align: top;\n",
-       "    }\n",
-       "\n",
-       "    .dataframe thead th {\n",
-       "        text-align: right;\n",
-       "    }\n",
-       "</style>\n",
-       "<table border=\"1\" class=\"dataframe\">\n",
-       "  <thead>\n",
-       "    <tr style=\"text-align: right;\">\n",
-       "      <th></th>\n",
-       "      <th>nodeId</th>\n",
-       "      <th>labels</th>\n",
-       "      <th>subject</th>\n",
-       "      <th>features</th>\n",
-       "    </tr>\n",
-       "  </thead>\n",
-       "  <tbody>\n",
-       "    <tr>\n",
-       "      <th>0</th>\n",
-       "      <td>31336</td>\n",
-       "      <td>Paper</td>\n",
-       "      <td>0</td>\n",
-       "      <td>[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>1</th>\n",
-       "      <td>1061127</td>\n",
-       "      <td>Paper</td>\n",
-       "      <td>1</td>\n",
-       "      <td>[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, ...</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>2</th>\n",
-       "      <td>1106406</td>\n",
-       "      <td>Paper</td>\n",
-       "      <td>2</td>\n",
-       "      <td>[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>3</th>\n",
-       "      <td>13195</td>\n",
-       "      <td>Paper</td>\n",
-       "      <td>2</td>\n",
-       "      <td>[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>4</th>\n",
-       "      <td>37879</td>\n",
-       "      <td>Paper</td>\n",
-       "      <td>3</td>\n",
-       "      <td>[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "    nodeId labels  subject                                           features\n",
-       "0    31336  Paper        0  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...\n",
-       "1  1061127  Paper        1  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, ...\n",
-       "2  1106406  Paper        2  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...\n",
-       "3    13195  Paper        2  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...\n",
-       "4    37879  Paper        3  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ..."
-      ]
```

Comment:
these outputs were considered useful in the past to give viewers a hint on how it will look.
Not clearly opposed to removing it, but was it needed for the testing?

Reply:
The testing doesn't do anything with the output atm. And even if it did, it would probably use output it produces when running the notebookes, instead of this cached output?

Reply:
I would personally lean towards no outputs as it makes our lives easier (see our GDS docs where we always fight with outputs).

Just on a note, we might want to add more asserts in the notebooks to make sure they dont change between versions

Reply:
> I would personally lean towards no outputs as it makes our lives easier (see our GDS docs where we always fight with outputs).

:+1: 

> Just on a note, we might want to add more asserts in the notebooks to make sure they dont change between versions

Yeah, I think we should have a work item for improving the notebook testing

Reply:
As you know I'm keen to keep the results from a docs reading perspective _and_ a (yet not completely defined) testing perspective, but if there is a strong opinion against keeping them, we can discard them.

I see two alternatives that we can try out:

- introduce `assert`s in the notebooks and use them for both testing and docs purposes
- use metadata to mark the output cells that we want to keep, so that the notebook is not polluted by unnecessary results but still contains output in "strategic" (documentation-oriented and/or test-oriented) places

I'll give it a try to both and raise a separate discussion.